src/dev/impl/DevToys/Helpers/UrlHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
```

Comment:
Are these necessary? I think only `System` is needed here.

Reply:
@veler , thank you good catch.  Would you be interested in adding 
```bash
dotnet_diagnostic.IDE0005.severity = error
```
to the `.editorconfig`? That way it would be clear to see:
![image](https://user-images.githubusercontent.com/74258557/230210629-1855558d-3078-4240-bcf9-c94593182302.png)

Reply:
Thanks for the suggestion. I'd rather avoid. In DevToys 2.0 branch, at least for now, we have some `using` that are wrongly marked as unnecessary, so I'm concerned it would be cause more pain for now. But I will keep that in mind for future evolution. :)